frontend/views/components/PageSection.vue

```diff
@@ -1,6 +1,8 @@
 <template lang='pug'>
 section.card
-  h3.title(v-if='title') {{ title }}
+  h3.title.c-title(v-if='title')
+    | {{ title }}
+    slot(name='cta')
```

Comment:
A heading (h3) should only contain text, no actions inside. These needs to be 2 different elements.

Reply:
Fixed!